docs/CH-Edgex-taf.rst

```diff
@@ -0,0 +1,149 @@
+#########
+EdgeX-taf
+#########
+
+ITAF Architecture Choices:
+
+* Well -defined Test Code Structure.
+* Test application code that narrates the use case scenarios are written using human-readable syntax with ROBOT tool.
+* Abstraction of test case code from test application code. Test case code are ROBOT Keywords defined in Python/ROBOT.
+* Separation of configuration data from test case application code. Configuration data can be in several formats including JSON, YAML, .cfg etc.
+* Separate test logs/reports from test application code.
+* Common test code utilities that are documented and catalogued for cross workgroup group reuse.
+* Embrace other test aid tools and test scope like JMETER, Selenium, postman, console-cli etc.
+* Integration with Jenkins facilitated by TAF Manager
+
+Overview
+========
+
+.. image:: images/Edgex-taf-overview.png
+    :scale: 50%
+    :alt: Edgex-taf Overview
+
+* Edgex-taf provides a well-defined project structure for the configuration and test cases, test scripts, and test report
+* Edgex-taf-common provides a place for reusable scripts
+
+How to Create an EdgeX-taf Project
+==================================
+
+To write the automation testing with edgex-taf framework, complete the following steps:
+
+1. Clone the edgex-taf project from EdgeX Foundry as a template::
+
+       git clone git@github.com:edgexfoundry/edgex-taf.git
```

Comment:
in edgex-taf I don't see that we have created submodule. Is this intentional ? If not I would prefer to do that 
if we have submodule, we can use --recursive flag to clone the submodule 
git clone --recursive git@github.com:edgexfoundry/edgex-taf.git

Reply:
We modify the edgex-taf-common as a python module.
You can install it via pip3
see https://github.com/edgexfoundry/edgex-taf-common/blob/master/README.md